whisper/cli/src/main.rs

```diff
@@ -19,6 +19,9 @@
 //! Spawns an Ethereum network instance and attaches the Whisper protocol RPCs to it.
 //!
 
+#![warn(missing_docs)]
+#![cfg_attr(feature = "cargo-clippy", deny(clippy, clippy_pedantic))]
```

Comment:
Can you elaborate on what this does? Disables slippy for this file only?

Reply:
I think `deny` turns warning into errors. The `!` applies this rule to the module (in this case only this file).

Reply:
Yeah, @ordian I think you are right.

But, in addition, `cargo-clippy` is only available on the nightly toolchain so these will be ignored if you use the original `rust-toolchain` file in Parity (assuming no overrides)!

cargo-clippy is essentially a modified verision of `rustc` that you run instead of rustc with additional lints i.e, instead of running rustc you run clippy!

However, it might be better to use `warn` instead of `deny`!